modules/data_release/README.md

```diff
@@ -1,16 +1,50 @@
-# Data Release Module
+# Data Release
 
-The Data Release Module can be used to easily distribute packaged data releases of your study.
 
-## Features:
+## Purpose
 
-- Upload new releases through the front-end web interface
-- Differentiate releases using (optionally unique) versioning tag
-- Assign permissions for each release to any specific user
+The Data Release module can be used to distribute packaged data releases 
+of the study to the analysis team. This module can be used by projects 
+which desires to publish based on the exact same set of data referred
+as data release. Along the lifespan of a study, several data releases might
+be made available as the dataset keeps growing.
 
-## Other notes:
 
-- Uploads are stored under the modules/data_release/user_uploads directory which can easily be symlinked to another location if necessary, but please create and make sure it is writable by your web server
-- Only superusers have permission to upload files and grant permissions
-- Remove permissions by deleting rows in the data_release_permissions table
-- Upload date will automatically be added during file upload
+## Intended users
+
+- Researchers and analysis teams
+- Data coordinator that will manage access to and upload of the data releases
```

Comment:
the tense changes between these two bullet points (but I can't remember my grammar well enough to remember what the tenses are called..)

I think the second should just be "Data coordinator that manages access to and uploads data releases"